html-forms/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+
+    <div>
+      <div>
+        <label for="email-address">Email</label>
+      </div>
+      <input type="text" name="email" id="email-address" />
+    </div>
+
+    <div><label for="why-coding-cool">Why is coding cool?</label></div>
+    <select name="options" id="why-coding-cool">
+      <option value="">Select an option</option>
+      <option value="interesting">Because it's interesting</option>
+      <option value="fun">Because it's fun</option>
+      <option value="powerful">Because it's powerful</option>
+    </select>
+
+    <div>Favorite Language (choose one)</div>
+
+    <input
+      type="radio"
+      name="favoriteLanguage"
+      value="html"
+      checked="checked"
+      id="html" /><label for="html">HTML</label>
```

Comment:
Why is this label on the same line as the end of the prior element? That is not proper HTML formatting 😬